Fix name of standard library in version 1.5

alan-j-hu opened the pull request.

alan-j-hu committed:
Fix name of standard library in version 1.5

jonaprieto commented:
The current file for the standard library is taken from committed file v1.5  of https://github.com/agda/agda-stdlib/blob/master/standard-library.agda-lib.
Did your installation fail?

alan-j-hu commented:
Yes, my installation failed.

```
$ apkg install standard-library
Installing (standard-library) from the index...
Installing from git: https://github.com/agda/agda-stdlib.git
Using temporary directory: /tmp/tmpud5wlzum
Downloading https://github.com/agda/agda-stdlib.git (223.8 KiB)
Git branch: master
|██████████████████████████████| 100%              
Library file detected: standard-library.agda-lib
Library version: v1.5
Adding /home/me/.apkg@agda-2.6.1.3/package-sources/standard-library-1.5@v1.5
error:  No file descriptor for the version v1.5 of standard-library-1.5.
error:  we couldn't install the version you specified.
Unsuccessfully installation (standard-library).
[!] /home/me/.agda/libraries already exists
Do you want to overwrite it? [y/N]: N
[!] /home/me/.agda/defaults already exists
Do you want to overwrite it? [y/N]: N
```

Installing 1.4 works:

```
$ apkg install standard-library --version v1.4
Installing (standard-library) from the index...
Installing from git: https://github.com/agda/agda-stdlib.git
Using temporary directory: /tmp/tmpvv6sjh00
Downloading https://github.com/agda/agda-stdlib.git (223.8 KiB)
Git branch: master
|██████████████████████████████| 100%              
Library file detected: standard-library.agda-lib
Library version: v1.4
Adding /home/me/.apkg@agda-2.6.1.3/package-sources/standard-library@v1.4
Successfully installed (standard-library@v1.4).
	Source code available on: /home/me/.apkg@agda-2.6.1.3/package-sources/standard-library@v1.4
[!] /home/me/.agda/libraries already exists
Do you want to overwrite it? [y/N]: Y
Updated (/home/me/.agda/libraries)
[!] /home/me/.agda/defaults already exists
Do you want to overwrite it? [y/N]: Y
Updated (/home/me/.agda/defaults)
```
